### ISSUE TITLE
Support for "automatic actions"?

### ISSUE THREAD

**Issue by waldner** [skupperproject/skupper#677]
Sorry, I don't even know how to call this. What I'd like to know is whether it's possible to have skupper perform specific actions (for example, create `Ingress` resources) when a service imported from an edge cluster appears in the core cluster.
If skupper cannot do it, I'd welcome some advice on what could be a recommended approach to do this (I'm thinking k8s custom controllers, but perhaps there are other ways). Thanks.

**Comment by ted-ross** [comment]
Hi,

Can you elaborate in a little more detail what it is you are trying to
accomplish?  Are you trying to create an Ingress for public access to a
service that exposes pods in a private edge?

-Ted

**Comment by waldner** [comment]
> Are you trying to create an Ingress for public access to a service that exposes pods in a private edge?

Yes, that could be a scenario, but I think the mechanism could be more general. I'd be happy to discuss the `Ingress` case only, for the sake of understanding. Thanks.
